### DISCUSSION TITLE
I don't like the "Popular among people you follow" suggestions

### DISCUSSION THREAD

**Discussion by pawamoy** [community/community#13241] [Other Feature Feedback, Questions, & Ideas]
These suggestions feel a bit too vague/general to me 😮 
I think I really don't need to see popular projects in my feed since they are, well, popular already.
I'd prefer "exploring" myself, sorting by trending or popular, rather than get these suggestions in my feed.
See some examples below.

![image](https://user-images.githubusercontent.com/3999221/159459619-0c1e2351-77a6-4b87-9f37-21ec57f55566.png)

I'm not at all into TypeScript.

---

![image](https://user-images.githubusercontent.com/3999221/159459667-d3020ebe-1685-451b-9f6a-d53115f81dd1.png)

I'm not at all into blockchain.

---

![image](https://user-images.githubusercontent.com/3999221/159459711-0792dbd5-c174-4051-b5bf-1d8f813ae8a0.png)

Just a very popular project I already heard of, don't need it in my feed.

---

![image](https://user-images.githubusercontent.com/3999221/159460190-c8ce29bf-57a5-482d-baac-ccadcca0a7b2.png)

Interesting but not (yet?) using Rust. If someone I follow starred it, it should already appear in my feed as such.

---

![image](https://user-images.githubusercontent.com/3999221/159462274-01082bb8-8e52-4023-b140-b7bcecd063ee.png)

That's nice but I already starred it.

**Comment by x86-39** [comment]
Yeah I think this would be better if it checked what languages, frameworks, etc YOU code in, not your friends. It can take inspiration from your friends stars but I don't really care for seeing a framework in a language I have no interest in learning.

**Comment by sntwandy** [comment]
That's improve can be interesting.

**Comment by jfullstackdev** [comment]
hey guys, there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
